pubpub/pubpub
# dev: add node labels for iframes

## Issue(s) Resolved

Resolves #2528 
Resolves #2780 

## Test Plan

- Visit the review app.
- Create a Pub and enable Iframe node labels in the Pub's settings.
- Embed an Iframe in the draft, e.g. https://pubpub-pipel-iframe-nod-ogbifp.herokuapp.com/pub/ptvkrnbn/draft
- Ensure the node is labeled "Iframe 1" (or whatever label you chose)

## Screenshots (if applicable)
<img width="96" alt="Screenshot 2023-07-18 at 5 12 29 PM" src="https://github.com/pubpub/pubpub/assets/6402908/8b8cfd8f-8fc2-4796-9636-49d00705997e">
